examples/express/README.md

```diff
@@ -44,59 +46,61 @@ To start collecting and exporting tracing data from the app it requires
 2. Configuration for the SDK.
-3. SDK initialization in [tracer.js](./tracer.js).
+3. SDK initialization before running the application.
 
-Once `.env` file is filled in with access token(see file [.env.example](./.env.example)) the app can be run by setting the environment variables and requiring SDK initialization just before the application code:
+All those steps are done for you in this example:
+
+1. Additional packages are already specified in the `package.json`.
+2. All the configuration is included in [an environment file `.env.collector`](./.env.collector).
+3. SDK intitialization in [tracer.js](./tracer.js).
+
+All that's left is to run it:
 
 ```shell
-npm run server
-# in a separate terminal:
-npm run client
+npm run server:collector
+# In a separate terminal:
+npm run client:collector
+
 ```
 
 See the exact commands in [package.json](./package.json).
-If unsure about the values to use for `.env.example` but familiar with configuring OpenTracing SDK, one can alternatively also use [.env.opentracing.example](./.env.opentracing.example) as the basis - the OTel configuration will be automatically derived from that in this example(see [utils.js](./utils.js) for the conversions).
 
 ### Instrumented via OTel SDK, Jaeger Exporter
 
-There's an [tracer setup](./tracer.jaeger.js) to showcase replacing the default OTLP/HTTP exporter with one that exports in Jaeger format. The configuration is similar to the plain OTel SDK setup, but
-
-1. `@opentelemetry/exporter-jaeger` package has to be installed and the factory initiating the Exporter has to be passed to `startTracing`;
-2. The endpoint(`OTEL_EXPORTER_JAEGER_ENDPOINT`) for Splunk's APM has to be replaced by one that accepts Jaeger formatted tracing data: `https://ingest.<realm>.signalfx.com/v2/trace`.
+There's an [tracer setup](./tracer.jaeger.js) to showcase replacing the default OTLP/gRPC Exporter with one that exports in Jaeger format. The configuration is similar to the plain OTel SDK setup, but the Exporter(`OTEL_TRACES_EXPORTER`) has to be replaced by the default Jaeger Exporter which is bundled with the SDK: `jaeger-thrift-http`.
 
 That's it! To run the example with the Jaeger Exporter:
 
 ```shell
 npm run server:jaeger
-# in a separate terminal:
+# In a separate terminal:
 npm run client:jaeger
 ```
 
-### Instrumented via OTel SDK, exporting to locally running collector
+### Instrumented via OTel SDK, exporting to Splunk APM using Jaeger Exporter
 
-Instead of sending telemetry data directly to Splunk APM API, one can also use OTel Collector to as a forwarder or an endpoint.
-There is an exmample [.env](./.env.collector) file included, which following commands use:
+Instead of using OTel Collector to forward the telemetry data to Splunk APM, one can also send it directly from the application.
+Once `.env` file is created(see file [.env.jaeger-splunk](./.env.jaeger-splunk) for an example) and the access token is substituted into the file. Run it using that:
```

Comment:
```suggestion
Instead of using the OTel Collector to forward telemetry data to Splunk APM, you can also send it directly from the application. Once the `.env` file is created (see [.env.jaeger-splunk](./.env.jaeger-splunk) for an example) and the access token is replaced in the file, run the collector using the following commands:
```